playbooks/roles/airship-deploy-tempest/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+deploy_tempest: true
+tempest_enable_cinder_service: true
+tempest_enable_glance_service: true
+tempest_enable_nova_service: false
```

Comment:
Nova and Neutron are still not supported? Is OSH team aware of this and a ticket filed?

Reply:
Nova and neutron are disabled by default in the upstream chart, so I just left it that way in this initial commit because I know tempest deploys and runs the image and volume tests successfully. I'm currently re-testing with them enabled and will change the defaults if everything checks out.